example/src/passthrough.rs

```diff
@@ -229,7 +229,7 @@ impl FilesystemMT for PassthroughFS {
         libc_wrappers::close(fh)
     }
 
-    fn read(&self, _req: RequestInfo, path: &Path, fh: u64, offset: u64, size: u32) -> ResultData {
+    fn read<F: FnOnce(Result<&[u8], libc::c_int>) -> ()>(&self, _req: RequestInfo, path: &Path, fh: u64, offset: u64, size: u32, result: F) {
```

Comment:
style nit: can you write this using the `result: impl FnOnce(....)` style instead? Also, it doesn't need the `-> ()` part.